src/test/java/de/tum/in/www1/artemis/service/DataExportCreationServiceTest.java

```diff
@@ -402,7 +410,8 @@ private void assertCorrectContentForExercise(Path exerciseDirPath, boolean cours
                         .isDirectoryContaining(path -> path.getFileName().toString().contains("plagiarism_case") && path.getFileName().toString().endsWith(FILE_FORMAT_CSV));
             }
         }
-        // only include automatic test feedback if the assessment due date is in the future
+        // only include automatic test feedback if the assessment due date is in the
+        // future
```

Comment:
Imo the comment doesn't need to be spread across two lines here
```suggestion
        // only include automatic test feedback if the assessment due date is in the future
```